debian/setup-user.sh

```diff
@@ -6,3 +6,5 @@ adduser --gecos user \
   user
 adduser user sudo
 echo "user:user" | chpasswd
+sudo adduser --system ggc_user
```

Comment:
@lukasz-laguna Greengrass is only one of the `Vitrobian` usecases. GGC-specific bits need not to be part of the generic image. Let's remove those:
- ggc user and group creation,
- sqlite3 installation

Link protection can be left as it can be considered a general improvement.

Reply:
The group and packages installation should be described in the GGC setup documentation (I believe that we already do that).